From: Robert Luberda <robert@debian.org>
Date: Mon, 7 Mar 2011 21:04:36 +0100
Subject: 0020 Mark Rcs_Id as unused

Fix `'Rcs_Id' defined but not used' warnings given by gcc -Wall.
Do the same for VersionID variable.
---
 buildhash.c             | 2 +-
 correct.c               | 2 +-
 defmt.c                 | 2 +-
 deformatters/defmt-c.c  | 2 +-
 deformatters/defmt-sh.c | 2 +-
 dump.c                  | 2 +-
 exp_table.c             | 2 +-
 fields.c                | 2 +-
 good.c                  | 2 +-
 hash.c                  | 2 +-
 icombine.c              | 2 +-
 ijoin.c                 | 2 +-
 ispell.c                | 2 +-
 lookup.c                | 2 +-
 makedent.c              | 2 +-
 parse.y                 | 2 +-
 sq.c                    | 2 +-
 term.c                  | 2 +-
 tgood.c                 | 2 +-
 tree.c                  | 2 +-
 unsq.c                  | 2 +-
 version.h               | 4 ++--
 xgets.c                 | 2 +-
 23 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/buildhash.c b/buildhash.c
index 4ed5f47..a826a8d 100644
--- a/buildhash.c
+++ b/buildhash.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: buildhash.c,v 1.76 2020-12-30 22:33:41-08 geoff Exp $";
 #endif
 
diff --git a/correct.c b/correct.c
index deb679c..e719228 100644
--- a/correct.c
+++ b/correct.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: correct.c,v 1.87 2021-01-09 13:12:35-08 geoff Exp $";
 #endif
 
diff --git a/defmt.c b/defmt.c
index d1b9d07..bd7990f 100644
--- a/defmt.c
+++ b/defmt.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: defmt.c,v 1.65 2020-12-30 22:33:41-08 geoff Exp $";
 #endif
 
diff --git a/deformatters/defmt-c.c b/deformatters/defmt-c.c
index 8c5192b..70119ca 100644
--- a/deformatters/defmt-c.c
+++ b/deformatters/defmt-c.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: deformat-c.c,v 1.3 2001/07/25 21:51:48 geoff Exp geoff $";
 #endif
 
diff --git a/deformatters/defmt-sh.c b/deformatters/defmt-sh.c
index 27d46fc..713fec6 100644
--- a/deformatters/defmt-sh.c
+++ b/deformatters/defmt-sh.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: deformat-sh.c,v 1.4 2001/07/25 21:51:48 geoff Exp geoff $";
 #endif
 
diff --git a/dump.c b/dump.c
index dc11b6b..0478cfe 100644
--- a/dump.c
+++ b/dump.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: dump.c,v 1.19 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/exp_table.c b/exp_table.c
index 941a9ee..35addb5 100644
--- a/exp_table.c
+++ b/exp_table.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: exp_table.c,v 1.4 2005/06/11 22:43:53 geoff Exp $";
 #endif
 
diff --git a/fields.c b/fields.c
index d793824..93a646a 100644
--- a/fields.c
+++ b/fields.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: fields.c,v 1.11 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/good.c b/good.c
index 466e2ff..d9179f4 100644
--- a/good.c
+++ b/good.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: good.c,v 1.51 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/hash.c b/hash.c
index 0f7f1e4..d2ad1cb 100644
--- a/hash.c
+++ b/hash.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: hash.c,v 1.25 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/icombine.c b/icombine.c
index 57efbd3..7ec699b 100644
--- a/icombine.c
+++ b/icombine.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: icombine.c,v 2.33 2005/04/20 23:16:32 geoff Exp $";
 #endif
 
diff --git a/ijoin.c b/ijoin.c
index edb18d1..3941533 100644
--- a/ijoin.c
+++ b/ijoin.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: ijoin.c,v 1.12 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/ispell.c b/ispell.c
index 9b509d0..c7058c9 100644
--- a/ispell.c
+++ b/ispell.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: ispell.c,v 1.161 2005/05/25 14:13:53 geoff Exp $";
 #endif
 
diff --git a/lookup.c b/lookup.c
index 4abf584..52ab1b0 100644
--- a/lookup.c
+++ b/lookup.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: lookup.c,v 1.54 2020-12-30 22:33:41-08 geoff Exp $";
 #endif
 
diff --git a/makedent.c b/makedent.c
index 3df9253..fb7a9e0 100644
--- a/makedent.c
+++ b/makedent.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
 	"$Id: makedent.c,v 1.62 2021-01-09 13:12:35-08 geoff Exp $";
 #endif
 
diff --git a/parse.y b/parse.y
index 8c35fae..7396e86 100644
--- a/parse.y
+++ b/parse.y
@@ -1,6 +1,6 @@
 %{
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: parse.y,v 1.67 2021-01-09 13:12:35-08 geoff Exp $";
 #endif
 
diff --git a/sq.c b/sq.c
index c931421..9b19e1f 100644
--- a/sq.c
+++ b/sq.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: sq.c,v 1.16 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/term.c b/term.c
index 24bebcc..799dc0b 100644
--- a/term.c
+++ b/term.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: term.c,v 1.57 2020-12-30 22:33:41-08 geoff Exp $";
 #endif
 
diff --git a/tgood.c b/tgood.c
index 5a71cb8..3cc753f 100644
--- a/tgood.c
+++ b/tgood.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: tgood.c,v 1.43 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/tree.c b/tree.c
index 073a6a6..f143787 100644
--- a/tree.c
+++ b/tree.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: tree.c,v 1.66 2005/04/14 14:43:46 geoff Exp $";
 #endif
 
diff --git a/unsq.c b/unsq.c
index 125f654..d698841 100644
--- a/unsq.c
+++ b/unsq.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: unsq.c,v 1.18 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
diff --git a/version.h b/version.h
index 746285e..535c69e 100644
--- a/version.h
+++ b/version.h
@@ -9,7 +9,7 @@
  * incorrect translation.
  */
 
-static char * Version_ID[] = {
+static char * Version_ID[] __attribute__ ((unused)) = {
     "@(#) International Ispell Version 3.4.02 08 Jan 2021",
     "@(#) Copyright (c), 1983, by Pace Willisson",
     "@(#) International version Copyright (c) 1987, 1988, 1990-1995, 1999,",
@@ -62,7 +62,7 @@ static char * Version_ID[] = {
     NULL
 };
 
-static char RCS_Version_ID[] =
+static char RCS_Version_ID[] __attribute__ ((unused)) =
     "$Id: version.h,v 1.62 2021-01-09 13:12:35-08 geoff Exp $";
 
 #endif /* VERSION_H_INCLUDED */
diff --git a/xgets.c b/xgets.c
index 44a7d51..293ec07 100644
--- a/xgets.c
+++ b/xgets.c
@@ -1,5 +1,5 @@
 #ifndef lint
-static char Rcs_Id[] =
+static char Rcs_Id[] __attribute__ ((unused)) =
     "$Id: xgets.c,v 1.28 2005/04/14 14:38:23 geoff Exp $";
 #endif
 
